pkg/operator/ceph/config/config_test.go

```diff
@@ -28,4 +28,20 @@ func TestNewFlag(t *testing.T) {
 	assert.Equal(t, NewFlag("b_key", "b"), "--b-key=b")
 	assert.Equal(t, NewFlag("c key", "c"), "--c-key=c")
 	assert.Equal(t, NewFlag("quotes", "\"quoted\""), "--quotes=\"quoted\"")
+	assert.Equal(t, NewFlag("osd_op_num_threads_per_shard_ssd", "2"), "--osd-op-num-threads-per-shard-ssd=2")
```

Comment:
We don't need those tests, just like @travisn said, we need to call makedeployment and count the number of args depending on the scenario.

Reply:
Removed!